templates/bootstrap/pyproject.toml.j2

```diff
@@ -13,6 +13,11 @@ title_format = "{version} ({project_date})"
 template = "CHANGES/.TEMPLATE.rst"
 issue_format = "`#{issue} <https://pulp.plan.io/issues/{issue}>`_"
 
+    [[tool.towncrier.type]]
+        directory = "deprecation"
+        name = "Deprecations"
+        showcontent = true
+
```

Comment:
Now i am wondering if this file should still be in the "bootstrap" section, because that makes it a mess to update in existing plugins.

Reply:
when I created a new type for pulp_installer, it stopped to recognize the old ones, I had to do it:
https://github.com/pulp/pulp_installer/blob/master/pyproject.toml#L36-L39
(Maybe I missed something there, I'm just pointing it out, for us to test this change)

Reply:
@fao89 I tested it out and you were right.

@mdellweg I'm a bit mixed since I agree that making this change to existing plugins will be a pain but also, I wonder if pyproject.toml will diverge across projects.

Reply:
This question (moving files) does not need to be answered here, obviously.

Reply:
@mdellweg it's definitely worth a discussion.